docs/usage.md

```diff
@@ -125,6 +125,11 @@
       - [`--freebayes_C`](#freebayesc)
       - [`--freebayes_g`](#freebayesg)
       - [`--freebayes_p`](#freebayesp)
+      - [`--pileupcaller_bedfile`](#pileupcallerbedfile)
+      - [`--pileupcaller_snpfile`](#pileupcallersnpfile)
+      - [`pileupcaller_majority_call`](#pileupcallermajoritycall)
```

Comment:
```suggestion
      - [`pileupcaller_majority_call`](#pileupcallermajoritycall)
```

You're missing the `--` for some of these parameters. Make sure to update the markdown headers themselves.